apps/pdc-frontend/package.json

```diff
@@ -12,7 +12,7 @@
     "acceptance-env": "STRAPI_PRIVATE_URL=https://pdc-api.cg-intern.acc.utrecht.nl/graphql STRAPI_PUBLIC_URL=https://pdc-api.cg-intern.acc.utrecht.nl next dev -p 8000",
     "prod-env": "STRAPI_PRIVATE_URL=https://pdc-api.cg-intern.utrecht.nl/graphql STRAPI_PUBLIC_URL=https://pdc-api.cg-intern.utrecht.nl next dev -p 3000",
     "prebuild": "yarn clean",
-    "codegen": "wait-on http://localhost:1337 && graphql-codegen",
+    "codegen": "wait-on http://localhost:1338 && graphql-codegen",
```

Comment:
@scar055 Dit lijkt me gewenste wijziging? Draai jij strapi op een andere port zelf?

Reply:
@AliKdhim87 had gezegd dat ik het beter op die port kon draaien maar ik het niet moest commiten dus ik had een vergissing gemaakt en is nu gefixed.